web/src/app/modules/views/side/modules/errors-warnings/components/errors-warnings.component.css

```diff
@@ -0,0 +1,7 @@
+.table {
+    margin: 0px;
+}
+
+.table th, .table td {
+    padding: 0.5rem;
```

Comment:
Is this the point where the padding around the table comes from? Please remove it, if so.